src/components/Api.js

```diff
@@ -0,0 +1,90 @@
+export default class Api {
+  constructor({ baseUrl, authorization }) {
+    this._baseUrl = baseUrl;
+    this._authorization = authorization;
+  }
+
+  getUserInfo() {
+    return fetch(`${this._baseUrl}/users/me`, {
+      headers: {
+        authorization: this._authorization,
+      },
+    });
+  }
+
+  getCardInfo() {
+    return fetch(`${this._baseUrl}/cards`, {
+      headers: {
+        authorization: this._authorization,
+      },
+    });
+  }
+
+  editUserInfo({ name, job }) {
+    return fetch(`${this._baseUrl}/users/me`, {
+      method: "PATCH",
+      headers: {
+        authorization: this._authorization,
+        "Content-Type": "application/json",
```

Comment:
According to the task you need to pass full `headers` into the constructor of `Api`. See here https://disk.yandex.ru/i/J93dbJp7eypXKQ . And in the code you need to use only: `headers: this._headers` (not to duplicate the code)